modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -10993,9 +11089,19 @@ type ScienceRequirements {
   mode: ScienceMode!
 
   """
-  Spectroscopy requirements
+  Requested exposure time mode.
   """
-  spectroscopy: SpectroscopyScienceRequirements!
+  exposureTimeMode: ExposureTimeMode
```

Comment:
`exposureTimeMode` was in `spectroscopy` but it is now on `ScienceRequirements`. This matches the table structure, where the mode is shared between the science modes

Reply:
We'll have to change our decoders some, but we can work around this.

Reply:
Yes I'll make the adjustments in explore

Reply:
I can do it. I'm working in explore, anyway.